<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWidget{
border-image:url(:/Assets/Image/Background1.jpg)
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>1024</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{
	background:rgb(248,248,255);
	border-top-style:solid;
	border-top-width:1px;
	border-top-color:rgba(220, 220, 220, 1);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnPre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnPre
{
	image: url(:/Assets/Icon/pre.png);
	background-color:transparent;
	border:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnNext
{
	image: url(:/Assets/Icon/next.png);
	background-color:transparent;
	border:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>15</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnPlay
    {
    image:url(:/Assets/Icon/pause.png);
    background-repeat:no-repeat;
    background-color:transparent;
    position:center center;
    border:transparent;
    }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlayMode">
    <property name="geometry">
     <rect>
      <x>814</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnPlayMode
    {
    image:url(:/Assets/Icon/loop.png);
    background-repeat:no-repeat;
    background-color:transparent;
    background-position:center center;
    border:transparent;
    }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="sliderProgress">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>20</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#sliderProgress
{
	background:transparent;
	border:15px solid rgb(248,248,255);
}
QSlider::add-page:Horizontal
{
	background-color: rgb(205,201,201);

}
QSlider::sub-page:Horizontal
{
	background-color: rgb(255, 20, 147);
}
QSlider::groove:Horizontal
{
	background:transparent;
	height:6px;
}
QSlider::handle:Horizontal
{
	height:	18px;
	width:	18px;
	margin: -6px;
	border-image:url(:/Assets/Icon/sliderHandle.png);
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnList">
    <property name="geometry">
     <rect>
      <x>963</x>
      <y>19</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnList
{
	image: url(:/Assets/Icon/list.png);
	background-color:transparent;
	border:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVol">
    <property name="geometry">
     <rect>
      <x>893</x>
      <y>24</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnVol
{
	image: url(:/Assets/Icon/vol.png);
	background-color:transparent;
	border:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lyricText">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>25</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>520</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="pagePlay">
    <widget class="QLabel" name="coverPic">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#coverPic
{
	image:url(&quot;:/Assets/Icon/non-music.png&quot;);
	background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="musicTitleLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>360</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(248, 248, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="musicAuthorLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>395</y>
       <width>360</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(248, 248, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="musicAlbumLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>435</y>
       <width>360</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(248, 248, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="LyricWidget" name="lyricWidget" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>600</width>
       <height>385</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageMusicList">
    <property name="styleSheet">
     <string notr="true">#pageMusicList
{
	background-color: rgb(248, 248, 255);
}</string>
    </property>
    <widget class="MusicListWidget" name="musicListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>1024</width>
       <height>465</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#musicListWidget
{
	background-color:rgba(220, 220, 220, 0.2);
	outline:none;
	border-image:transparent;
}
QListWidget::item:selected:active
{
	background:rgb(255, 20, 147);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSortSinger">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>9</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnSortSinger
{
	image: url(:/Assets/Icon/singer.png);
	background-color:transparent;
	border:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSortTitle">
     <property name="geometry">
      <rect>
       <x>231</x>
       <y>7</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnSortTitle
{
	image: url(:/Assets/Icon/title.png);
	background-color:transparent;
	border:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSortDuration">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>11</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnSortDuration
{
	image: url(:/Assets/Icon/duration.png);
	background-color:transparent;
	border:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>7</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnAdd
{
	image: url(:/Assets/Icon/folder.png);
	background-color:transparent;
	border:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNeaten">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>7</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnNeaten
{
	image: url(:/Assets/Icon/neaten.png);
	background-color:transparent;
	border:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSweep">
     <property name="geometry">
      <rect>
       <x>358</x>
       <y>7</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnSweep
{
	image: url(:/Assets/Icon/sweep.png);
	background-color:transparent;
	border:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSlider" name="sliderVol">
   <property name="geometry">
    <rect>
     <x>884</x>
     <y>320</y>
     <width>50</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sliderVol
{
background-color:rgb(248,248,255);
min-width:20px;
max-width:20px;
border:15px solid rgb(248,248,255);
}
QSlider::add-page:vertical
{
	background-color: rgb(255, 20, 147);
}
QSlider::sub-page:vertical
{
	background-color: rgb(205,201,201);
}
QSlider::groove:vertical
{
	background:transparent;
	width:8px;
}
QSlider::handle:vertical
{
	height:	18px;
	width:	18px;
	margin: -5px;
	border-image:url(:/Assets/Icon/sliderHandle.png);
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>984</x>
     <y>15</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnQuit
{
	image: url(:/Assets/Icon/quit.png);
	background-color:transparent;
	border:transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>btnQuit</zorder>
  <zorder>sliderVol</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">musiclistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>LyricWidget</class>
   <extends>QWidget</extends>
   <header location="global">lyricwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
